<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QMainWindow" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>127</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>420</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="serverNameLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="serverNameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Server name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serverNameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="serverPortLayout" stretch="0,0">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="serverPortLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Server port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serverPortEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>onConnectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverNameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>onServerSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverPortEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>onServerSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectClicked()</slot>
  <slot>onServerSettingsChanged()</slot>
 </slots>
</ui>
